erpnext/non_profit/doctype/membership/membership.py

```diff
@@ -50,4 +50,33 @@ def on_payment_authorized(self, status_changed_to=None):
 			self.load_from_db()
 			self.db_set('paid', 1)
 
+@frappe.whitelist()
+def create_membership(name, email, membership_type, from_date, to_date, currency, amount, paid):
```

Comment:
So many mandatory args?

Reply:
Set a few to defaults, but most of these args are required